Google.Cast/source/Google.Cast/Extensions.cs

```diff
@@ -122,7 +122,7 @@ public partial class Logger
 		}
 
 
-		public void Log (string function, params string[] message)
+		public void Log (string function, params string [] message)
```

Comment:
Should this be public void Log (string function, string format, params object [] args)

Then we do a string.Format (format, args) and only pass the result to the native method